testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -19,7 +21,12 @@ - (void)setUp {
 
 - (void)tearDown {
   if (self.flutterViewController) {
-    [self.flutterViewController removeFromParentViewController];
```

Comment:
The test bug was here. 

The flutterViewController was presented, not added as a child vc. Sequencing 2 consecutive tests that present a flutter vc blocks on each other.